libtailscale/backend.go

```diff
@@ -212,7 +212,7 @@ func (a *App) runBackend(ctx context.Context) error {
 			if cfg.rcfg != nil && state >= ipn.Starting {
```

Comment:
Is it idiomatic go practice to algebraically compare 'enums' like this instead of:

switch val {
   case a,b,c,e,f: doThing()
   case d: doOtherThing()
}

//add g, compiler failure

This seems very dangerous...